client/my-sites/plans/controller.jsx

```diff
@@ -38,7 +38,8 @@ export function plans( context, next ) {
 					? context.query.addDomainFlow === 'true'
 					: undefined
 			}
-			domainAndPlanPackage={ context.query.domainAndPlanPackage }
+			domainAndPlanPackage={ JSON.parse( context.query.domainAndPlanPackage ?? false ) }
```

Comment:
`Non-blocking / nitpick`: 

I find this more readable `Boolean( context.query.domainAndPlanPackage )`

Reply:
Thanks! That's is a good idea to explicitly check for the `"true"` case rather than doing parsing and conversion that obscures the readability of the code 👍